README.md

```diff
@@ -72,26 +72,31 @@ Then run:
 * [Install pre-commit globally](https://pre-commit.com/#installation)
 * From your checkout directory run `pre-commit install`
 
-## Run locally
-`flask db migrate`
-`flask run`
+## Run App or Unit Tests Locally
+Local Docker stack must be up-to-date and running with `docker compose up` in order to use the app/tests locally:
+https://github.com/communitiesuk/funding-service-design-post-award-docker-runner
 
-## Updating database migrations
-Whenever you make changes to database models, please run:
-`flask db migrate`
-This will create the migration files for your changes in ./db/migrations. Please then commit and push these to github so that the migrations will be run in the pipelines to correctly upgrade the deployed db instances with your changes.
+Make migrations to ensure Docker PostgreSQL container is up-to-date:
```

Comment:
I think this should be "Apply migrations..."

Reply:
changed, thanks